### PR TITLE
Buff the borg's laser gun.

### PR DESCRIPTION
**Pull request by r4d6**
## About The Pull Request
This buff the laser gun, at @Trilbyspaceclone's request, that security borgs get with the illegal module, due to it being weaker than a 'weak' energy beam.

### CODE REVIEW

File: code/modules/projectiles/guns/energy/laser.dm [hunk 1]
```diff
@@ -39,7 +39,13 @@
 /obj/item/weapon/gun/energy/laser/mounted/cyborg
 	name = "integrated \"Cog\" lasgun"
 	desc = "A Greyson Positronic design, cheap and widely produced. In the distant past - this was the main weapon of low-rank police forces, billions of copies of this gun were made."
-	recharge_time = 10 //Time it takes for shots to recharge (in ticks)
+	icon = 'icons/obj/guns/energy/retro.dmi'
+	icon_state = "retro"
+	recharge_time = 4 //Time it takes for shots to recharge (in ticks)
+	damage_multiplier = 1
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/energy/laser.dm:45]:
```suggestion
	damage_multiplier = 0.8
```

Comment (Trilbyspaceclone) [reply code/modules/projectiles/guns/energy/laser.dm:45]: This is a 300~% increse in damage. + 30 AP making it around 350~% damage as it was before.

**Comment by ThePainkiller** [comment]
Alternative title: Borg gets a free Titanica laser canon but somewhat weaker (28 burn with 25 armor pen unless I'm mistaken)

**Comment by r4d6** [comment]
> Alternative title: Borg gets a free Titanica laser canon but somewhat weaker (28 burn with 25 armor pen unless I'm mistaken)

Well, I was asked by Trilby to buff it, so I buffed it.
Previously the gun dealth 14 burn damage, that's weaker than the weak laser.

**Pull request merged**